.circleci/config.yml

```diff
@@ -256,9 +263,16 @@ jobs:
           condition: << parameters.needs_postgres_source >>
           steps:
             - run: git submodule update --init --depth 1
+      - restore_cache:
+          keys:
+            - v1-python-deps-{{ checksum "poetry.lock" }}
       - run:
           name: Install deps
           command: ./scripts/pysync
+      - save_cache:
+          key: v1-python-deps-{{ checksum "poetry.lock" }}
+          paths:
+            - /home/circleci/.cache/pypoetry/virtualenvs
```

Comment:
What do you think about using `poetry config virtualenvs.create false --local` instead?

Reply:
Then only cache dir changes to pip's default I guess (or to some other dir inside poetry's .cache) Either way is ok to me, what's your preference? Personally I manage virtualenvs via pyenv so I have this setting locally

Another option might be to cache just *.whl files, not the whole venv, but current approach works well and wheel handling requires some boilerplate, so I would rather stick to current approach

Reply:
> Either way is ok to me, what's your preference?

I don't have a strong preference either, just had a thought that the origin of this path might not be as obvious as the local per-project env. I guess we'll change it if it ever becomes a problem.